### PR TITLE
error on invalid usage

### PR DESCRIPTION
**Pull request by sigmaSd**

### CODE REVIEW

**Comment by sigmaSd** [comment]
maybe this work for urls, I'll have to check that

**Comment by sigmaSd** [comment]
ah it already doesn't

File: cli.ts [hunk 1]
```diff
@@ -181,6 +182,29 @@ function _task(task: string): void {
   }
 }
 
+function _ensureJsFiles(paths: string[]) {
+  let errors = 0;
+  for (const path of paths) {
+    if (
+      path.includes(".") && !(path.endsWith(".js") || path.endsWith(".ts"))
```
Comment (hasundue) [inline cli.ts:189]: Could you 
- include `.jsx` and `.tsx` as well?
- use `extname` from `std/path` to check file extensions?

Comment (sigmaSd) [reply cli.ts]: done

**Review by hasundue — APPROVED** [review]
Thanks!

**Pull request merged**